themes/ddbasic/sass/components/class.scss

```diff
@@ -0,0 +1,794 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?